doc/dependencies.md

```diff
@@ -7,11 +7,11 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 | --- | --- | --- | --- | --- | --- |
 | Berkeley DB | [4.8.30](https://www.oracle.com/technetwork/database/database-technologies/berkeleydb/downloads/index.html) | 4.8.x | No |  |  |
 | Boost | [1.71.0](https://www.boost.org/users/download/) | [1.64.0](https://github.com/bitcoin/bitcoin/pull/22320) | No |  |  |
-| Clang |  | [5.0+](https://releases.llvm.org/download.html) (C++17 support) |  |  |  |
+| Clang |  | [8.0+](https://releases.llvm.org/download.html) (C++17 support with P0083R3 proposal) |  |  |  |
```

Comment:
Most Clang usage will still be on top of libstdc++, and it's still possible to compile master with Clang 5 and libstdc++ 7+. i.e on Bionic.

Reply:
> ... libstdc++ 7+. i.e on Bionic.

To be precise, [7.5.0](https://packages.ubuntu.com/bionic/libstdc++-7-dev).

Reply:
> with Clang 5 ... on Bionic.

Do you mean [clang 6.0](https://packages.ubuntu.com/bionic/clang)?

Reply:
> Do you mean clang 6.0?

No. I mean [clang-5.0](https://packages.ubuntu.com/bionic/clang-5.0).

Reply:
Thanks! Updated.